relayer/cmd/generate_beacon_data.go

```diff
@@ -186,54 +189,57 @@ func generateBeaconTestFixture(cmd *cobra.Command, _ []string) error {
 			return err
 		}
 
+		inboundFixturesOnly, err := cmd.Flags().GetBool("inbound_fixtures_only")
+
 		// generate InitialUpdate
 		initialSyncScale, err := s.GetCheckpoint()
 		if err != nil {
 			return fmt.Errorf("get initial sync: %w", err)
 		}
 		initialSync := initialSyncScale.ToJSON()
-		err = writeJSONToFile(initialSync, fmt.Sprintf("%s/%s", pathToBeaconTestFixtureFiles, "initial-checkpoint.json"))
-		if err != nil {
-			return err
+		if !inboundFixturesOnly {
+			err = writeJSONToFile(initialSync, fmt.Sprintf("%s/%s", pathToBeaconTestFixtureFiles, "initial-checkpoint.json"))
+			if err != nil {
+				return err
+			}
```

Comment:
I think it would be cleaner and clearer to just create a new command than adding all these `if !inboundFixturesOnly {`.

Reply:
https://github.com/Snowfork/snowbridge/pull/1158/commits/e5f5092af6c9ac9ff1c935341971ad3dca0ae30d